<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminlugar</class>
 <widget class="QDialog" name="Adminlugar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 195, 221);</string>
  </property>
  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>691</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jamrul</family>
     <pointsize>36</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);
font: 75 36pt &quot;Jamrul&quot;;
color: rgb(73, 60, 107);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>580</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver lugar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>481</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);
</string>
   </property>
   <property name="title">
    <string>Comentarios/Valoraciones</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea_Coments">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_comentarios">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>453</width>
         <height>433</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_null">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;DejaVu Sans Mono&quot;;
color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>No hay comentarios de tu negocio</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>481</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(243, 243, 243);
</string>
   </property>
   <property name="title">
    <string>Reportes</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_reportes">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>455</width>
         <height>435</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label_null_2">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;DejaVu Sans Mono&quot;;
color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>No hay reportes de tu negocio</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 154, 154);</string>
   </property>
   <property name="text">
    <string>Eliminar Lugar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cerrar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar Sesi√≥n</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
